README.md

```diff
@@ -117,7 +117,23 @@ Check out how in the INSTALL section.
 
 3) Edit config.cfg to suit your needs (see [configuration](#configuration)). `acme-dns` will read the configuration file from `/etc/acme-dns/config.cfg` or `./config.cfg`
 
-4) Run acme-dns. Please note that acme-dns needs to open a privileged port (53, domain), so it needs to be run with elevated privileges.
+4) If your systemd has systemd, you can optionally install acme-dns as a service so that it will start on boot and be tracked by systemd. This also allows us to add the `CAP_NET_BIND_SERVICE` capability so that acme-dns can be run by a user other than root.
```

Comment:
I'm terribly sorry for unprofessional review comment, but I couldn't resist.


![systemd](https://user-images.githubusercontent.com/5235109/40574713-9567112c-60df-11e8-86f6-42eceabb07ed.jpg)

Reply:
Haha it was worth it! I had typed "system" and "systemd" too many times...